daemon/daemon_main.go

```diff
@@ -894,15 +894,17 @@ func initEnv(cmd *cobra.Command) {
 	if !option.Config.EnableIPv4 && !option.Config.EnableIPv6 {
 		log.Fatal("Either IPv4 or IPv6 addressing must be enabled")
 	}
-	if err := kvstore.Setup(option.Config.KVStore, option.Config.KVStoreOpt); err != nil {
-		addrkey := fmt.Sprintf("%s.address", option.Config.KVStore)
-		addr := option.Config.KVStoreOpt[addrkey]
+	go func() {
```

Comment:
I am always wary of how we change Daemon bootstrapping, specifically if there is anything later that implicitly depends on the kvstore being ready and dependent upon the fact that setup was done synchronously before.